core/src/main/java/org/polypheny/db/sql/SqlInsert.java

```diff
@@ -177,5 +178,32 @@ public void unparse( SqlWriter writer, int leftPrec, int rightPrec ) {
     public void validate( SqlValidator validator, SqlValidatorScope scope ) {
         validator.validateInsert( this );
     }
+
+
+    public SchemaType getSchemaType() {
+        // this breaks previously written test if it tries to use catalog
+        // TODO DL: change
+        //if ( Catalog.isUnitTest ) {
+        return SchemaType.RELATIONAL;
+        //}
+        /*
+        try {
+            // here we use default value TODO DL: change
+            ImmutableList<String> names = ((SqlIdentifier) targetTable).names;
+            if ( names.size() == 1 ) {
+                return Catalog.getInstance().getSchema( "APP", "public" ).schemaType;
+            } else if ( names.size() == 2 ) {
+                return Catalog.getInstance().getSchema( "APP", names.get( 0 ) ).schemaType;
+            } else {
+                return Catalog.getInstance().getSchema( names.get( 0 ), names.get( 1 ) ).schemaType;
+            }
+
+        } catch ( UnknownSchemaException | UnknownDatabaseException e ) {
+            throw new RuntimeException( e );
+        }
+
+         */
```

Comment:
This looks a bit messy...

Reply:
This stems from the an initial approach and was not longer necessary, removed.